books/schemas/types.py

```diff
@@ -1,19 +1,47 @@
+from typing import Optional
+
 import graphene # type: ignore
+from graphene.types.datetime import Date # type: ignore
+
+from books.entities import Author, Book
+
+
+class BookAuthorType(graphene.ObjectType):
+    id = graphene.ID()
 
+    first_name = graphene.String()
+    last_name = graphene.String()
+    full_name = graphene.String()
+
+    def resolve_full_name(author: Author, info) -> str:
+        return f"{author.first_name} {author.last_name}"
 
-### TODO
-class BookType(graphene.ObjectType):
+
+class RelatedBookType(graphene.ObjectType):
```

Comment:
RelatedBookType? Like category? like gore, love story etc?

Reply:
Same i think add category model/file is good idea :D

Reply:
1. I mean rather thinking about the continuation of a book for example if the book has two parts then the `related_book` field is the next part of the first part of the book (if this makes any sense)
2. Yeah good idea. I will add that to a model :)